frontend/tsconfig.json

```diff
@@ -7,16 +7,25 @@
     "skipLibCheck": true,
     "esModuleInterop": true,
     "types": ["node", "jest", "@testing-library/jest-dom"],
-    /* Bundler mode */
     "moduleResolution": "bundler",
     "allowImportingTsExtensions": true,
     "resolveJsonModule": true,
     "isolatedModules": true,
     "noEmit": true,
     "jsx": "react-jsx",
     "sourceMap": true,
-    "inlineSources": true
+    "inlineSources": true,
+    "baseUrl": ".",
+    "paths": {
+      "@src/*": ["src/*"],
+      "components/*": ["src/components/*"],
+      "utils/*": ["src/utils/*"],
+      "lib/*": ["src/lib/*"],
+      "pages/*": ["src/pages/*"],
+      "@test/*": ["__tests__/src/*"]
+    }
```

Comment:
Is it possible to have baseUrl set to `src` and in paths keep `@tests` alias only?

Reply:
Hey ! ..i tried doing so ...so when base url is src the test folder which is out side the src folder  is not recognising the paths that we are  mentioning .

Reply:
Yeah in the latest commits I tried to do it ...but jest tests were failing for such configuration they weren't able to point to src .. so when I explicitly mentioned the src in ts config along with @test and also added to the jest config ...
Then everything was running fine .

Reply:
Tbh ...it was difficult to debug the actual findings of the errors .

Reply:
Yes, keeping code base nice and tidy isn't easy. It requires some extra time and skills (also stubbornness sometimes) to get it done.

I've updated the PR with how I wanted imports to be reconfigured -- please review when you get a chance.
Thank you!

Reply:
@arkid15r LGTM

Reply:
we have switched to baseUrl --. pattern again !

Reply:
> @arkid15r LGTM

This code doesn't work in the app even though the tests are fine

Reply:
@arkid15r Yeah ...your right !...let me check